contracts/facets/GettersFacet.sol

```diff
@@ -22,6 +22,169 @@ import {EnumerableSet} from "@openzeppelin/contracts/utils/structs/EnumerableSet
 contract GettersFacet is IGetters {
     using EnumerableSet for EnumerableSet.AddressSet;
 
+    /// @notice This function is used as a helper for front-end implementation
+    /// @param termId The term id for which the summary is being requested
+    /// @return term complete term object
+    /// @param joinPositions available positions if any
+    /// @param joinAmounts the minimum security deposit for each position from joinPositions
+    function getTurnGroupRelatedSummary(
+        uint termId
+    )
+        external
+        view
+        returns (
```

Comment:
Couldn't we make structs for this inside the getters facet instead? Specific structs for this that would contain the return values of each summary function. For example the struct of the fund would contain:

            fund.initialized,
            fund.currentState,
            fund.stableToken,
            fund.beneficiariesOrder,
            fund.fundStart,
            fund.fundEnd,
            fund.currentCycle,
            fund.totalAmountOfCycles